typings/index.d.ts

```diff
@@ -836,12 +836,12 @@ declare module 'discord.js' {
 		public readonly displayName: string;
 		public guild: Guild;
 		public readonly id: Snowflake;
-		public readonly joinedAt: Date | null;
-		public joinedTimestamp: number | null;
+		public readonly joinedAt: Date;
+		public joinedTimestamp: number;
```

Comment:
Should be marked as null, due to https://github.com/discordjs/discord.js/commit/13bfceb83cf771d1d56d28fd474d824f9a8884db.

Reply:
Updated, thanks for the explanation